.changeset/biome_logs_a_warning_in_case_a_folder_contains_biomejson_and_biomejsonc_and_it_will_use_biomejson_by_default.md

```diff
@@ -0,0 +1,5 @@
+---
+cli: patch
+---
+
+# Biome logs a warning in case a folder contains `biome.json` and `biome.jsonc`, and it will use `biome.json` by default.
```

Comment:
Actually this was not a bug. This was an intended feature by the implementer.
However, it is not so important ^^

Reply:
The functionality is still the same, however this was an implicit behaviour. Now the warning makes it clearer. It still makes sense to have this as patch